docs/content/stable/deploy/public-clouds/aws/manual-deployment.md

```diff
@@ -316,8 +316,8 @@ for ip in $ALL_NODES; do \
    echo =======$ip=======; \
    ssh -i $PEM $ADMIN_USER@$ip \
       "cd ~/yb-software; \
-       curl -k -o yugabyte-${YB_VERSION}-linux.tar.gz \
-         https://downloads.yugabyte.com/yugabyte-${YB_VERSION}-linux.tar.gz"; \
+       curl -k -o -L yugabyte-${YB_VERSION}-linux.tar.gz \
+         https://software.yugabyte.com/yugabyte-${YB_VERSION}-linux.tar.gz"; \
```

Comment:
This seems out of date, the URL doesn't include the `releases/` component and there is no arch encoded in the file name.  It should be updated to match the same pattern as in `install-software.md`